js/custom/cognitoUserIdentity.js

```diff
@@ -118,10 +118,12 @@ function signInFunction() {
                 var str = decoded.name + " " + decoded.family_name;
                 var id = decoded.email+"|"+str;
                 alert("id constracted " + id);
+                alert("test cognito test: "+ decoded.language)
                 sessionStorage.setItem("userName", str);
                 sessionStorage.setItem("userId",id);
                 sessionStorage.setItem("idToken",idToken);
                 sessionStorage.setItem("userIsLoggedIn","true");
+                sessionStorage.setItem("language","tigrinya");
```

Comment:
Since we expect a value to be sent to us from the server here indicating the language, it may be better to check the decoded language and default to tigrigna if no value is sent. So, something like this:

```suggestion
                sessionStorage.setItem("language", decoded.language ? decoded.language : "tigrinya");
```